README.md

```diff
@@ -8,71 +8,117 @@
 </p>
 
-### Install
-```bash
-$ npm install --global @asyncapi/cli
+<p align="center">
+<a href="https://github.com/asyncapi/cli/blob/master/LICENSE"><img alt="GitHub license" src="https://img.shields.io/github/license/asyncapi/cli"></a>
+<a href="https://github.com/asyncapi/cli/actions/workflows/if-nodejs-pr-testing.yml">
+<img src="https://github.com/asyncapi/cli/actions/workflows/if-nodejs-pr-testing.yml/badge.svg" alt="PR testing - if Node project"  />
+</a>
+<a href="https://www.npmjs.com/package/@asyncapi/cli">
+<img alt="npm" src="https://img.shields.io/npm/dw/@asyncapi/cli">
+</a>
+
+</p>
+
+## Table of contents
+
+1. [Getting Started](#getting-started)
+    - [Installation](#installation)
+2. [Usage](#usage)
+3. [API Reference](#api-reference)
+    - [`validate`](#validate)
+    - [`context`](#context)
+4. [Contributing](#contributing)
+
+## Getting Started
+To run `@asyncapi/cli`, ensure that you have NodeJs >=v10. [Install NodeJs via package manager](https://nodejs.org/en/download/package-manager/).
+
+### Installation
+As of now the only way to install `asyncapi/cli` is using NPM. If you have NodeJs installed, it is most likely that you have NPM installed as well. 
+
 ```
+npm install -g @asyncapi/cli
+```
+
+This installs the cli globaly on your system allowing you to run it from anywhere. If you want to install it locally, just remove the `-g` flag. 
+
+[back to top ⏫](#table-of-contents)
+
 
-### Getting Started
-Go ahead and run command `asyncapi --help` to get complete help for using CLI. If having doubt about any particular command do run `asyncapi <command> --help` to get help for that command.  
+## Usage
 
-### CLI
-Help string for all the supported commands
+As of now `@asyncapi/cli` supports only validation of spcification file, but is under development for mode features.
 
-- #### `asyncapi --help`
+We have well documented help commands so just run 
+
+```
+asyncapi --help 
 ```
-USAGE
+[back to top ⏫](#table-of-contents)
 
-asyncapi [options] [command]
 
-OPTIONS
 
--h, --help display help for command
--v, --version output the version number
+## API Reference
+
+### `validate`
 
-COMMANDS
+**USAGE**
 
-validate validate asyncapi file 
-context  Manage contexts
 ```
+asyncapi validate <spcPath | context-name> [options]
+```
+If you have current context set then 
+```
+asyncapi validate
+```
+
+**OPTIONS** 
 
-- #### `asyncapi validate --help`
 ```
-USAGE
+-h, --help Display help for command
+-w, --watch Enable watch mode (not implemented yet)
+```
 
-asyncapi validate [options]
+[back to top ⏫](#table-of-contents)
 
-OPTIONS
+## `context`
+
+Context is what makes it easier for you to work with multiple Asyncapi Files. You can add multiple different files to a context. This way you do not have to pass file as a input everytime you use any command. You can also set a default context, so neither you have to pass in filepath or context name as a input. 
+
```

Comment:
```suggestion
**Context** makes it easier for you to work with multiple AsyncAPI Files. You can add multiple different files to a context so that you don't have to pass the file as an input every time you use any command. You can also set a default context so that you don't have to pass in either file paths or context names as an input. 
```